release-team/role-handbooks/docs/README.md

```diff
@@ -58,6 +70,26 @@ General time requirements for leads and shadows are:
 - Between 1 and 2 hours a week to attend the majority of Release Team (weekly) and Burndown meetings (daily during Code Freeze), subject to time zone appropriateness
 - Up to 1 hour weekly to attend [SIG Docs meetings](https://github.com/kubernetes/community/tree/master/sig-docs#meetings) for status reports
```

Comment:
Can we add a bullet point to highlight the final release notes PR creation and reaching out to the sigs (via slack and email) to ask them to review the final draft?
 ```suggestion
- Up to 1 hour weekly to attend [SIG Docs meetings](https://github.com/kubernetes/community/tree/master/sig-docs#meetings) for status reports
- Create [known issues issue](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/release-notes/known-issues-bucket.md) in kubernetes/kubernetes to capture known issues for the release
- Send [an email to SIG-leads](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/release-notes/sig-leads-email.md) to ensure major changes for their SIGs are accurately reflected in the release notes
- Send a slack message to the sig channels to ensure major changes for the SIGs are accurately reflected in the release notes 
```